src/components/scenes/GettingStartedScene.tsx

```diff
@@ -95,10 +107,10 @@ export const GettingStartedScene = (props: Props) => {
   const swipeOffset = useSharedValue(0)
 
   // Route helpers
-  const getNewAccountRoute = (createAccountType: CreateAccountType): InitialRouteName => {
+  const getNewAccountRoute = (): InitialRouteName => {
```

Comment:
Hey, can't these be converted to `InitialRouteName` types instead of `(): InitialRouteName`?